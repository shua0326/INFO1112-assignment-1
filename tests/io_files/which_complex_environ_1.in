which ls cd
var PATH ""
which ls pwd
var PATH "/usr/sbin"
which ls echo
