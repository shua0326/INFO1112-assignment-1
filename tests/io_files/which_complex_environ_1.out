>> /usr/sbin/ls
cd: shell built-in command
>> >> ls not found
pwd: shell built-in command
>> >> /usr/sbin/ls
/usr/sbin/echo
>> 
